.github/workflows/build-and-test.yaml

```diff
@@ -33,11 +33,19 @@ jobs:
             override: true
 
       - name: Run cargo check
+        if: "!startsWith(matrix.rust, 'nightly')"
```

Comment:
It would be good to check everything (benches, bins, examples, tests) on nightly anyways.
```suggestion
```

Reply:
It's odd, apparently I don't fully understand the `-Z features=dev_dep`. Let me try again to achieve something like this:

> By running cargo check -Z features=dev_dep, that can be detected (require nightly).

Maybe it's enough to literally run it like that, without additional flags, in addition to what's always run.
Commit is coming up.